src/3k-mapper.xml

```diff
@@ -1154,6 +1169,170 @@ end</script>
                     <packageName></packageName>
                     <regex>^tmap addalias &quot;(.+)&quot; (.+)$</regex>
                 </Alias>
+                <Alias isActive="yes" isFolder="no">
+                    <name>Rename an area</name>
+                    <script>--renames current area
+local Name = matches[2]
+local cArea = getRoomArea(tmap.lastId)
+setAreaName( cArea, Name )</script>
+                    <command></command>
+                    <packageName></packageName>
+                    <regex>^tmap renamearea (.+)$</regex>
+                </Alias>
+                <Alias isActive="yes" isFolder="no">
+                    <name>Connect all stub in area</name>
+                    <script>local aid = getRoomArea(tmap.lastId)
+local rooms = getAreaRooms(aid)
+for i,v in pairs(rooms) do
+	local stubs = getExitStubs(v)
+	if stubs and type(stubs) == 'table' then
+		for j,k in pairs(stubs) do
+			connectExitStub(v,k)
```

Comment:
As I note in comment 6 [bug 1114594](https://bugs.launchpad.net/mudlet/+bug/1114594) this only works where the first argument is the fromRoomId and the second argument is a direction code (1 to 12) or a specific exit command string.  In this case it IS so WILL work. :relieved: